app/models/volunteer_op.rb

```diff
@@ -14,7 +14,6 @@ class VolunteerOp < ActiveRecord::Base
   scope :reachskills,          -> { where(source: 'reachskills') }
   scope :remote_only,          -> { where.not(source: 'local') }
 
```

Comment:
Sorry, I'll avoid it in the future :) . I'll redo code as it was, but in case if it keeps showing a lot of changed lines I can also name methods which were changed. Actually, this model was not changed very much. I simply added one more method and refactored another to avoid duplication.